packages/patternfly-4/react-core/src/layouts/Grid/Grid.docs.js

```diff
@@ -14,5 +14,5 @@ export default {
   enumValues: {
     gridSpans: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
   },
-  examples: [Simple, WithGutters, Overrides, GridPlayground]
+  examples: [Simple, WithGutters, Overrides, { example: GridPlayground, live: false }]
```

Comment:
To follow the other configs for each example it might be better to put this as a static property on the examples class instead. This will keep consistency as well as make it obvious in the example file if it supports live edit.

Reply:
Am wondering if we shouldn't keep the examples "pure" so that consumers could copy/paste without additional code that doesn't do anything for them.

Reply:
that wouldn't be a bad idea. I originally planned to use babel to strip all the demo/example data, but honestly, that seems a bit overly complicated. It might be nice to transfer all examples to use this object structure and drop all of the static properties. (title, description, etc) That can be done in a separate PR though.

Reply:
I'll make it a static for now as you've suggested, we can rethink this approach in the future

Reply:
@dmiller9911 agree we can do a switch over of static props to objects in *.docs.js files in a separate PR